bin/oneoff/backfill_data/backfill_address_line1_in_schools

```diff
@@ -33,14 +33,16 @@ failure_count = 0
 
 open(err_file, 'w') do |err|
   CSV.foreach(tmp_file.path, {col_sep: "\t", headers: true, quote_char: "\x00"}) do |row|
-    begin
-      school = School.find_by_id(row['NCESSCH'].to_i.to_s)
-      school[:address_line1] = row['LSTREE']
-      school.save!
-      success_count += 1
-    rescue
-      err << row
-      failure_count += 1
+    school = School.find_by_id(row['NCESSCH'].to_i.to_s)
+    unless school.nil?
+      begin
+        school[:address_line1] = row['LSTREE']
+        school.save!
+        success_count += 1
+      rescue
+        err << row
+        failure_count += 1
+      end
```

Comment:
Do we expect any errors? Should we rescue more specific error types and log the message?

Reply:
Actually, there were errors in the prior run but that's because there was no corresponding school entries. It's not clear what the original dataset was from or if schools were added to the list afterwards. Once I write the more generic merge script, I'll handle specific errors that come up.